package.json

```diff
@@ -15,18 +15,18 @@
     "civicsource"
   ],
   "peerDependencies": {
-    "eslint": ">= 3 < 5"
+    "eslint": "5.x",
+    "prettier": "1.x"
```

Comment:
Why is this a peer dependency now?

Reply:
It was made a peerDependency of eslint-plugin-prettier, but it appears that's been the case since eslint-plugin-prettier@2.x. Also since it's that package's peerDependency maybe it should be unspecified/a devDependency & consuming applications should specify?